index.html

```diff
@@ -10,7 +10,6 @@
 </head>
 
 <body>
-<script src="js/scripts.js"></script>
```

Comment:
I moved this to the bottom of the body tag. Best practice, as I mentioned earlier, is to do so so that all of your elements are rendered in the page before the JS can run. While this isn't what was causing the problem, it's good to get into that habit.